deploy/catalog_resources/courier/kiecloud-operatorsource.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: operators.coreos.com/v1
 kind: OperatorSource
 metadata:
-  name: kiecloud-operators
+  name: kogitocloud-operator
```

Comment:
The file should be renamed, too.

Reply:
Yeah... I was thinking about it... I don't know why I didn't rename it :-1: